### ISSUE TITLE
Implement right click functionalities in the sketchbook

### ISSUE THREAD

**Issue by Akarshit** [processing/processing#4253]
When the sketchbook is opened basic functions should be provided on right clicking the sketches like
* Open
* Delete
* Rename
* Run
* Copy
* Paste
* Properties
* Move(?)

I will try to implement these features and create a PR. 
If some has any other feature in mind i would be happy to incorporate it.

**Comment by PARAG00991** [comment]
@Akarshit I agree completely with you that functions should be provided for sketches in sketchbook. What I was thinking to add to this list is, there should be 2 options for **open** :-

- Open in same window
- Open in new window

At present , when we click a sketch in sketchbook, by default ,it gets opened in new window of processing. It could be implemented such as to give user two options, either to open any stored sketch in a new tab in the same working window or to open it in a completely new window.
